eng/tools/suppressions/test/e2e.test.ts

```diff
@@ -0,0 +1,38 @@
+import { expect, test } from "vitest";
+import { getSuppressions } from "../src/suppressions.js";
+import { join } from "path";
+
+async function getTestSuppressions(path: string, tool: string = "TestTool") {
```

Comment:
I do like writing out the function return types explicitly everywhere (except some most basic cases like `const foo = 3`, if you don't mind that.

Reply:
I think it's a bit wasteful of keystrokes, but I agree it makes the code easier to read outside the IDE.  Can we enforce this with ESLint or similar, once we onboard all eng projects?

Reply:
Yes we totally can :D